modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -14,90 +14,167 @@ Object Store provides these APIs:
+* xref:osv2-guide.adoc#send-data-to-an-object-store[Store a key-value pair in the object store]. 
+* <<get-bearer-token,Get the bearer token>> for an Anypoint Platform user using the Access Management API:
 +
-You need the access token to authenticate with the Object Store v2 Stats API.
+You need the bearer token to access all Object Store v2 APIs.
+* <<get-org-env-id,Get the ORG_ID and ENV_ID>>.
 +
-See
-https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/Authentication/[Authentication] in the 
-https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/home/[Access Management API]. 
-* Get the organization ID (ORG_ID).
+You need the organization ID and environment ID for operations with all Object Store v2 APIs.
+* <<get-region-id,Get the REGION_ID>>.
 +
-You need the organization ID for operations with the Object Store v2 API and Object Store v2 Stats API.
-+
-See the `/api/me` endpoint in the
-https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/home/[Access Management API].
-* Get the environment ID (ENV_ID).
+You need the region ID for operations with the Object Store v2 Stats API.
+* <<get-store-id,Get the STORE_ID>>.
 +
-You need the environment ID for operations with the Object Store v2 API and Object Store v2 Stats API.
+You need the store ID for operations with the Object Store v2 Stats API.
+
+
+[[get-bearer-token]]
+== Get a Bearer Token to Use with the Object Store v2 APIs
+
+If MFA is enabled::
+
+Use a connected app to get the bearer token:
+
+. xref:osv2-connected-apps.adoc[Configure a connected app].
 +
-See the `/api/organizations/ORG_ID/environments` endpoint in the
-https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/home/[Access Management API].
-* Get the region ID (REGION_ID).
+Assign the connected app the Organization Administrator scope to access the organization and environment queries.
 +
-You need the region ID for operations with the Object Store v2 Stats API.
+. From the *Connected Apps* page, click *Copy Id* and *Copy Secret* for the app you just created.
 +
-See the `/api/v1/organizations/ORG_ID/regions` endpoint in the
-https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/object-store-v2/minor/1.0/pages/home/[Object Store v2 API] and 
-<<Example: Retrieve a List of Regions>>.
-* Get the store ID (STORE_ID).
+Use this client ID and client secret to get the bearer token required to authenticate with the Object Store v2 API.
+. Run the following command to get the bearer token from the connected app client ID and client secret, 
```

Comment:
. To get the bearer token from the connected app client ID and client secret, run the following command:
+
<command>
+ 
Replace AUTH_SERVER_HOST with the URL of the authorization server, for example, `\https://anypoint.mulesoft.com/`.

Reply:
I'm trying to find a way to make it shorter. I had to read it a couple of times.